pytest/DynamoDBAdapter_test.py

```diff
@@ -0,0 +1,11 @@
+import os
+from dynamo_backupper.DynamoDBAdapter import DynamoDBAdapter
+
+def test_scan():
+    # Create the adapter instance
+    db_adapter= DynamoDBAdapter(os.environ.get('REGION'))
+
+    # Try to fetch data, if it blows up let it so the stack get logged into logs
+    data = db_adapter.scan(os.environ.get('TABLE_NAME'))
+
+    assert(data)
```

Comment:
Sound test but violates 'self-contained' aspect of unit test. Looks like there might be nifty way to run a minimal dynamodb locally for testing. Perhaps [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html#docker)? This would be a good way to dev without consuming our RCU's/WCU's...

Reply:
Merging as-is on account of KT.